language/move-lang/src/parser/ast.rs

```diff
@@ -347,18 +347,14 @@ pub enum ModuleAccess_ {
 }
 pub type ModuleAccess = Spanned<ModuleAccess_>;
 
-#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Hash)]
-pub enum Kind_ {
-    // Kind representing all types
-    Unknown,
-    // Linear resource types
-    Resource,
-    // Explicitly copyable types
-    Affine,
-    // Implicitly copyable types
-    Copyable,
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Hash)]
+pub enum Ability_ {
```

Comment:
Will it be good to re-use the `enum vm::file_format::Ability`?, i.e.,
```rust
pub type Ability_ = vm::file_format::Ability;
```

I don't have a strong preference but I guess this would also save a bit of code duplication on the `impl Ability_ {}` part?

Reply:
It would not be good to reuse that. Too many special cases for impls/traits/debugging/display/etc